pkgs/top-level/all-packages.nix

```diff
@@ -8432,6 +8432,8 @@ with pkgs;
   };
   libkrb5 = krb5Full.override { type = "lib"; };
 
+  languageMachines = import ../development/libraries/languagemachines/packages.nix { inherit callPackage; };
```

Comment:
That's a very good point! I will make frog available at the top level. I don't think its dependencies are used a lot outside frog, so I think it doesn't make sense to put everything in the already big `all-packages.nix`.

Is there a way to also include my test without polluting `all-packages.nix`? (It's a simple `runCommand` derivation that makes sure frog works as expected without a mutable `frog.out`, lingering builder state, etc.)